vendor/openfx/OCIOProcessor.h

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#ifndef INCLUDED_OFX_OCIOPROCESSOR_H
+#define INCLUDED_OFX_OCIOPROCESSOR_H
+
+#include "ofxsImageEffect.h"
+#include "ofxsMultiThread.h"
+#include "ofxsProcessing.H"
+
+#include <OpenColorIO/OpenColorIO.h>
+namespace OCIO = OCIO_NAMESPACE;
+
+class OCIOProcessor : public OFX::ImageProcessor 
+{
+protected:
+    OFX::Image * _srcImg;
+
+    OCIO::ConstCPUProcessorRcPtr _cpuProc;
+
+public:
+    OCIOProcessor(OFX::ImageEffect & instance)
+        : OFX::ImageProcessor(instance)
+        , _srcImg(0)
+        , _cpuProc(nullptr)
```

Comment:
[Minor] Smart pointers do not need to be initialized with nullptr. That's done.
In C++11 or higher, the naked pointer initialization should use `nullptr` instead of `0`.

Reply:
.. in fact, you could change line 17 to `OFX::Image * _srcImg = nullptr;`

Reply:
Fixed. Thanks!